### DISCUSSION TITLE
Use TailwindCSS in Gatsby theme plugin

### DISCUSSION THREAD

**Discussion by magne** [tailwindlabs/tailwindcss#2968] [Help]
I would like to use TailwindCSS (v. 2.x) in a theme plugin. My theme works fine, importing the `src/styles/globals.css` in `gatsby-browser.js`. Also, I get no warnings when I `gatsby build` or `gatsby develop`. 

When I include the theme in a site, my `gatsby-browser.js` import seems gone (a theme fontFamily is not found), and during `gatsby build`I get a warning warning about purging not configured.

Is it possible to build a theme plugin that completely encapsulates the use of TailwindCSS?

**Comment by simonswiss** [comment]
Hey!

Are you able to share some code on how you setup the theme?

I am not super familiar with Gatsby themes just yet, but I'd love to take a look 👍
